frontend/src/app.jsx

```diff
@@ -1,58 +1,57 @@
 import { useEffect, useState } from "react";
-
+import Task from "./components/task.jsx";
 
 export default function App() {
-  const [task, settasks] = useState([]);
+  const [tasks, setTasks] = useState([]);
```

Comment:
Just a naming change, from `task` and `settask` to `tasks` and `setTasks`, because:
- Plural form because this state variable can contain multiple tasks.
- From `settasks` to `setTasks` because the convention in JS is using camelCase for naming variables.